logproof/src/fields.rs

```diff
@@ -70,16 +80,20 @@ pub struct SealQ128_4096 {}
 
 /**
  * The configuration type for q modulus SEAL BFV uses with 128-bit security
- * an lattice dimension 4096.
+ * an lattice dimension 2048.
  *
  * # Remarks
  *
- * SEAL uses Q =
- * 0xffffee001, 0xffffc4001, 0x1ffffe0001
+ * SEAL uses Q = 0x3fffffff000001.
+ *
+ * This can be derived by running
+ * `CoefficientModulus::bfv_default(2048, SecurityLevel::TC128)`
+ * or by running the underlying SEAL function
+ * `CoeffModulus::BFVDefault(2048, sec_level_type::tc128)`
  */
 #[derive(MontConfig)]
 #[generator = "3"]
-#[modulus = "4611686014132420865"]
+#[modulus = "18014398492704769"]
```

Comment:
Hmm... 2048 and 1024 shouldn't be affected by the special modulus, so not sure why this changed.

Reply:
The numbers just didn't match what was in the file; I am not sure why. The encrypted data also does not exceed the new modulus specified here, so I assume it was just a typo somewhere.

Reply:
It seems that both values had an extra set of `0x01` appended:

```
degree 1024        2048
old    0x7e0000101 0x3fffffff00000101
new    0x7e00001   0x3fffffff000001
```

Reply:
Confirmed by saving the modulus value and performing the encryption manually that these new moduli are the correct ones (for 1024 and 2048).